src/main/scala/grizzled/file/Implicits.scala

```diff
@@ -34,7 +34,7 @@ object Implicits {
       if (file.exists)
         Success(true)
       else
-        Failure(new FileDoesNotExistException(s""""$file" does not exist."""))
+        Failure(new FileDoesNotExistException(s""""${file.toString}" does not exist."""))
```

Comment:
curious about this and the other similar diffs, why is this necessary/desirable?

Reply:
The `[wartremover:StringPlusAny] Implicit conversion to string is disabled`-wart triggers on scala 2.13.0-RC1, which maybe it shouldn't? On scala 2.12.8 it does not.

Reply:
ah. hmmm....

I think I personally wouldn't expect or want the wart to trigger inside a string interpolator, but that's just a hunch, not sure what the wart's author's intentions might have been

Reply:
I filed an issue here: https://github.com/wartremover/wartremover/issues/447